File: Spring/Backend/src/main/java/com/example/Backend/Model/Project.java
package com.example.Backend.Model;

public class Project {

    private int id;

    private int managerId;

    private String name;

    private String description;

    private String members;

    public Project(int id, int managerId, String name, String description, String members){
        this.id = id;
        this.managerId = managerId;
        this.name = name;
        this.description = description;
        this.members = members;
    }

    public int getId() {
        return id;
    }

    public int getManagerId() {
        return managerId;
    }

    public void setManagerId(int managerId) {
        this.managerId = managerId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getMembers() {
        return members;
    }

    public void setMembers(String members) {
        this.members = members;
    }




}

File: Spring/Frontend/src/app/model/project.ts
export class Project {

  public id: number;
  public managerId: number;
  public name: string;
  public description: string;
  public members: string;

  constructor(id: number, projectManagerId: number, name: string, description: string, members: string) {
    this.id = id;
    this.managerId = projectManagerId;
    this.name = name;
    this.description = description;
    this.members = members;
  }

  toString(): string {
    return "Name: " + this.name + " Description: " + this.description + " Members: " + this.members + " Project Manager ID: " + this.managerId;
  }


}
